cernopendata/modules/fixtures/data/records/phenix-emcal-finding-pi0s-and-photons.json

```diff
@@ -0,0 +1,259 @@
+[
+  {
+    "abstract": {
+      "description": "This record contains datasets from the Electromagnetic Calorimeter (EMCal) of the PHENIX detector. It aims to present a few basic techniques of identifying π0-s and photons using that device. The data is provided in the form of ROOT files containing small samples (Ntuples) of reconstructed p+Au data taken with the PHENIX detector in 2015 at √sNN=200 GeV. The Gamma-Gamma Ntuples (ggntuple) and Single Cluster Ntuples (gnt) are provided. Both minimum bias and EMCal-triggered data are included. <em>These samples were created for educational and training purposes only and are not at all usable for deriving any real physics results.</em> An example code working with the data is attached as ROOT macros. A detailed guide is also provided."
+    },
+    "accelerator": "BNL-RHIC",
+    "authors": [
+      {
+        "name": "David, Gabor",
+        "orcid": "0000-0003-4958-1347"
+      }
+    ],
+    "collaboration": {
+      "name": "PHENIX collaboration"
+    },
+    "collections": [
+      "PHENIX-Derived-Datasets"
+    ],
+    "dataset_semantics": [
+      {
+        "description": "Event centrality",
+        "type": "FIXME",
```

Comment:
Since these files are ROOT Ntuples, where only float is allowed, all variables are float

Reply:
Thanks, updated.